### PR TITLE
docs: MaaS introduction enhancement

### PR DESCRIPTION
**Pull request by antongisli**
## Describe the Change

This PR aims to make a more succinct and understandable introduction to the pcg 

## Review Changes

💻 [Preview](https://deploy-preview-1272--docs-spectrocloud.netlify.app/clusters/data-center/maas/install-manage-maas-pcg)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1272#" title="Author self-approved">antongisli</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @antongisli. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md [hunk 1]
```diff
@@ -13,10 +13,19 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. The primary function of the PCG is to facilitate connectivity between Palette and MAAS, even if MAAS is behind a NAT gateway or a firewall. The PCG "dials home" to Palette and establishes a permanent connection, traversing any NAT gateways or firewalls, rather than Palette attempting to connect to MAAS directly and being blocked by a firewall. In this way, the function of the PCG is somewhat similar to that of a reverse proxy.
+
+At a very high level, the following will occur during a successful MAAS PCG installation:
+- use a PCG installer docker image that you will run on a laptop, workstation, or jump box
+- provide information to the installer so that it can connect both to a local MAAS installation and to a Palette account
+- the installer will use MAAS to obtain machine(s), and then install a PCG on them
+- the PCG will now facilitate all communication between Palette and MAAS, enabling Palette to create new clusters on machines provided by MAAS
```
Comment (ritawatson) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:22]:
```suggestion
- The PCG then facilitates all communication between Palette and MAAS, enabling Palette to create new clusters on machines that MAAS provides.
```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md [hunk 2]
```diff
@@ -13,10 +13,19 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
 
 # Overview 
 
-The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster based on your requirements for high availability (HA).  
+The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. The primary function of the PCG is to facilitate connectivity between Palette and MAAS, even if MAAS is behind a NAT gateway or a firewall. The PCG "dials home" to Palette and establishes a permanent connection, traversing any NAT gateways or firewalls, rather than Palette attempting to connect to MAAS directly and being blocked by a firewall. In this way, the function of the PCG is somewhat similar to that of a reverse proxy.
+
```
Comment (ritawatson) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:17]: I would move lines 24 and 26 here as an intro to your process bullet list below:

"Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation, or jump box. You can use the installer on any Linux x86-64 system with a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint. 

You can set up the PCG as a single- or three-node cluster based on your requirements for high availability (HA)."

**Review by ritawatson — APPROVED** [review]
This is a good addition to the doc - thanks Anton. I suggest moving a couple lines ahead of the process list being added. I also suggested a few wording and styling changes.

LGTM with suggestions.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1272#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1272#pullrequestreview-1409278749" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ritawatson** [comment]
That's fine Anton. It's your call.

**Comment by cloudmaniac** [comment]
FYI, I have a couple of notes from when I tried the MAAS PCG deployment last week. I'll add my commits tomorrow or Friday.

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md [hunk 3]
```diff
@@ -13,10 +13,19 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+At a high level, the following occurs during a successful MAAS PCG installation:
+- Run a PCG installer Docker image on a laptop, workstation, or jump box.
```
Comment (cloudmaniac) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:19]:
```suggestion
- Run the PCG installer on a laptop, workstation, or jump box.
```

**Comment by karl-cardenas-coding** [comment]
@cloudmaniac we should remove the steps in the UI and instead link to the docs page. No one is really maintaining the UI text so it could lead to outdated instructions.

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md [hunk 4]
```diff
@@ -106,17 +115,17 @@ If you have already installed the PCG and are experiencing issues that you want
...
 4. The installer generates the PCG configuration file from information you provide in step **3**.
 
     <br />
 
-    The installer needs access to your Palette account and to one (no HA) or three (HA) machines in your MAAS cluster. If you select one machine in Palette, then you need one in MAAS. Likewise, if you select three machines in Palette, you need three in MAAS. The MAAS machines must have internet access and be in a ready state.
+    The installer needs access to your Palette account and to your MAAS environment. Additionally, one (no HA) or three (HA) machines must be in ready state and have internet access in MAAS. If you select one machine in step 3, then you need one in MAAS. Likewise, if you select three machines in step 3, you need three in MAAS.
     <br />
 
-5. The installer installs to the MAAS machine(s) and uses the configuration file to build a new cluster to host the PCG application. 
+5. The installer installs to the MAAS machine(s) and uses the configuration file to build a new cluster to host the PCG application.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:128]:
```suggestion
5. The installer installs the MAAS machines and uses the configuration file to build a new cluster to host the PCG application.
```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md [hunk 5]
```diff
@@ -371,7 +384,13 @@ Follow these steps to resize a single-node gateway to three nodes.
 
 4. Change the number of nodes to 3.
 
-Two new nodes will be created in the cluster.
+Two new nodes will be added to the PCG cluster.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:387]:
```suggestion
Two new nodes will be added to the PCG cluster.

<br />

```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md [hunk 6]
```diff
@@ -221,7 +234,7 @@ When you have entered all the configuration values, the installer saves the gate
 
 <InfoBox>
 
-The ``/opt/spectrocloud`` folder is volume mapped to the ``/tmp`` folder on the host machine.
+The ``/opt/spectrocloud`` folder is volume mapped to the ``/tmp`` folder on the installer.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:237]:
```suggestion
The **/opt/spectrocloud** folder is volume mapped to the installer's **/tmp** folder.
```

File: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md [hunk 7]
```diff
@@ -199,19 +208,23 @@ The installer does not work with SSO or Social sign on credentials. You must use
...
+    - Availability Zone
     - Resource Pool
     - One node (no HA) or three nodes (HA)
 
-9. Ensure the MAAS server has one or more machines in the **Ready** state for the chosen Availability Zone and Resource Pool combination.
+<WarningBox>
+
+Ensure the MAAS server has one or more machines in the **Ready** state for the chosen Availability Zone and Resource Pool combination.
+
+</WarningBox>
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md:227]:
```suggestion
  <br />
  
  <WarningBox>

  Ensure the MAAS server has one or more machines in the **Ready** state for the chosen Availability Zone and 
  Resource Pool combination.

  </WarningBox>
```

**Pull request merged**